app/src/main/java/com/lastaoutdoor/lasta/ui/screen/map/MapScreen.kt

```diff
@@ -122,7 +125,10 @@ fun MapScreen(
     updateSelectedMarker: (Marker) -> Unit,
     clearSelectedItinerary: () -> Unit,
     selectedZoom: Float,
-    updateSelectedItinerary: (Long) -> Unit,
+    selectedMarker: Marker?,
+    selectedItinerary: MapItinerary?,
+    markerList: List<Marker>,
```

Comment:
Is this markerList always non-null (initialized)?

Reply:
yes! In the concept, markerList is eithe an empty list or a List<Marker> with different values